plugins/extraction/ocaml.ml

```diff
@@ -494,9 +494,9 @@ let pp_one_ind prefix ip_equiv pl name cnames ctyps =
             (fun () -> spc () ++ str "* ") (pp_type true pl) typs)
   in
   pp_parameters pl ++ str prefix ++ name ++
-  pp_equiv pl name ip_equiv ++ str " =" ++
-  if Int.equal (Array.length ctyps) 0 then str " unit (* empty inductive *)"
-  else fnl () ++ v 0 (prvecti pp_constructor ctyps)
+  pp_equiv pl name ip_equiv ++
+  if Int.equal (Array.length ctyps) 0 then mt ()
```

Comment:
This is not the empty type, this is merely an abstract type. You should write `= |` as [per the doc](https://ocaml.org/manual/emptyvariants.html#s:empty-variants).